velodyne_pointcloud/launch/VLP-32C_points.launch

```diff
@@ -0,0 +1,51 @@
+<!-- -*- mode: XML -*- -->
+<!-- run velodyne_pointcloud/CloudNodelet in a nodelet manager for an VLP-32C -->
+
+<launch>
+
+  <!-- declare arguments with default values -->
+  <arg name="calibration" default="$(find velodyne_pointcloud)/params/VeloView-VLP-32C.yaml"/>
```

Comment:
It seems like this parameter file is missing from this PR.

Reply:
Thanks for catching this, @trainman419.

@ASDeveloper00 - Please add this file ASAP.

Reply:
Nevermind. I realized I have it as well. @trainman419  - It has been added.